docs/root/faq/api/control_plane_version_support.rst

```diff
@@ -30,6 +30,11 @@ typical rollout sequence might look like:
 4. Support for v2 is removed in the management server. The management server moves to v3 exclusively
    internally and can support newer fields.
 
+5. Client can send discovery request with v2 resource type url and process discovery response with 
```

Comment:
I think this belongs in a slightly different part of this advice. This enumerated list gives how to do a rollout with a management server that supports mixed v2/v3 versions. Your PR is adding support so that a control plane operator can rollout a v3-only management server in the presence of v2-only management servers and have the clients work with both. It's a more flexible way to do rollouts.

Reply:
@chaoqin-li1123 I think the entire two paragraphs you added belong in a separate section below, not in the list of steps for servers which both support v2 and v3. There is essentially guidance for servers with both v2/v3, which is steps 1-4, and another section for servers that only have v2 XOR v3 during a rollout.

Reply:
Thanks, will change the doc.